package.json

```diff
@@ -110,25 +110,25 @@
     "jest-dev-server": "^10.0.0",
     "mkdirp": "^3.0.1",
     "pako": "^2.1.0",
-    "prettier": "^3.2.5",
+    "prettier": "^3.3.2",
     "puppeteer": "^15.0.0",
     "release-it": "^17.2.1",
-    "rollup": "^4.17.2",
+    "rollup": "^4.18.0",
     "rollup-plugin-bundle-size": "^1.0.3",
     "serve": "^14.2.3",
-    "terser": "^5.31.0",
-    "ts-jest": "^29.1.2",
+    "terser": "^5.31.1",
+    "ts-jest": "^29.1.4",
     "ts-json-schema-generator": "^1.5.0",
     "typescript": "~5.4.5",
-    "vega-cli": "^5.28.0",
+    "vega-cli": "^5.29.0",
```

Comment:
In the past we didn't bump the dependency requirement for Vega until necessary. I don't mind to bump though.